R/calcCapacity.R

```diff
@@ -156,7 +185,23 @@ calcCapacity <- function(subtype) {
     output[getRegions(WEOcap),getYears(WEOcap),getNames(WEOcap)] <- WEOcap[getRegions(WEOcap),
                                                                            getYears(WEOcap),
                                                                            getNames(WEOcap)]
-
+    
+    #    ***CG: fix CHA gas power capacities: 97 GW by September 2020 (Oxford Institute for Energy Studies:
```

Comment:
I don't really understand these scripts - just wondering why you need to input the data twice?

Reply:
this was the legacy from when we created a switch for wind offshore, so there are two similar copies of input data for capacity, one with and one without wind offshore.. So both input files are created, and when offshore wind is off in REMIND, it is using a file without wind offshore in it. I thought about making the with windoffshore one default, but then I remember we probably need the switch for DIETER coupling (without the very exogenously defined offshore wind production). This is probably a tangent issue tho - What do you think?

Reply:
> this was the legacy from when we created a switch for wind offshore, so there are two similar copies of input data for capacity, one with and one without wind offshore

Which is no reason to define identical data twice in two different locations.  Define it once, then drop it into `output` at two points.  Else you will happily change the data for one file at some point in the future, wondering why your change does not affect the other file, which you are looking at.

Reply:
thanks - I cleaned up the redundant scripts - could you check quickly? thx